app/src/main/java/edu/uc/groupProject/topten/dto/TopTenList.kt

```diff
@@ -19,9 +19,11 @@ data class TopTenList(
     var id:Int, var title:String, var description:String="", var isActive:Boolean,
     var category:String /* Possible category dto? */, var publishDate:Date,
      var expireDate: Date
+
```

Comment:
I like the comment you made here! Makes clear the thought-process you are going through and can spark discussion among the team.